src/main/java/com/ldtteam/structurize/placement/AbstractBlueprintIterator.java

```diff
@@ -9,117 +9,49 @@
 import java.util.Collections;
 import java.util.function.Supplier;
 
-/**
- * This is the blueprint iterator.
- * It's a helper class used to track the progress of one iteration over the structure.
- */
-public abstract class AbstractBlueprintIterator
+public abstract class AbstractBlueprintIterator implements IBlueprintIterator
 {
-    /**
-     * The position we use as our uninitialized value.
-     */
-    public static final BlockPos NULL_POS = new BlockPos(-1, -1, -1);
-
-    /**
-     * The Structure position we are at. Defaulted to NULL_POS.
-     */
-    protected final BlockPos.MutableBlockPos progressPos = new BlockPos.MutableBlockPos(-1, -1, -1);
-
-    /**
-     * The size of the structure.
-     */
-    protected final BlockPos size;
```

Comment:
protected and public fields are for sure binary breaking

Reply:
`protected` *might* be fine, in this particular case, because I don't think any other mod than Structurize was extending AbstractBlueprintIterator. But better safe than sorry